docs/devguide/changelog.rst

```diff
@@ -2,6 +2,73 @@ Changelog
 =========
 
 
+Parsl 1.0.0
+-----------
+
+Released on June <FILL>th, 2020
+
+Parsl v1.0.0 includes 57 closed issues and 232 pull requests with contributions (code, tests, reviews and reports) from:
+
+Akila Ravihansa Perera @ravihansa3000, Aymen Alsaadi @AymenFJA, Anna Woodard @annawoodard,
+Ben Clifford @benclifford, Ben Glick @benhg, Benjamin Tovar @btovar, Daniel S. Katz @danielskatz,
+Daniel Smith @dgasmith, Douglas Thain @dthain, Eric Jonas @ericmjonas, Geoffrey Lentner @glentner,
+Ian Foster @ianfoster, Kalpani Ranasinghe @kalpanibhagya, Lindsey Gray @lgray, Logan Ward @WardLT,
+Lyle Hayhurst @lhayhurst, Mihael Hategan @hategan, Rajini Wijayawardana @rajiniw95, @saktar-unr,
+Tim Shaffer @trshaffer, Tom Glanzman @TomGlanzman, Yadu Nand Babuji @yadudoc and,
+Zhuozhao Li @ZhuozhaoLi
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+* `WorkQueueExecutor`: a new executor that integrates functionality from `Work Queue <http://ccl.cse.nd.edu/software/workqueue/>`_ is now available.
+
+  * WorkQueueExecutor is now in Beta.
+  * WorkQueueExecutor now supports function tagging and resource specification
+  * Support for resource specification kwarg `issue#1675 <https://github.com/Parsl/parsl/issues/1675>`_
+
+* `HighThroughputExecutor` will expose worker information via environment variables: `PARSL_WORKER_RANK` and `PARSL_WORKER_COUNT`
+* Separate htex worker log directories by block id `issue#1508 <https://github.com/Parsl/parsl/issues/1508>`_
+* Remove App decorator deprecated in 0.8 `issue#1539 <https://github.com/Parsl/parsl/issues/1539>`_
+* Significant progress in an ongoing effort to add type-checking to Parsl internal components.
+* IPyParallelExecutor is no longer a supported executor `issue#1565 <https://github.com/Parsl/parsl/issues/1565>`_
+* Improvements to caching mechanism <Ask ben for help> including ability to mark certain arguments to be
+  not counted for memoization.
+* `File` objects.
+* Updates to documentation
+  * <KYLE> anything we want to call out ?
+* New launcher: `WrappedLauncher` for launching tasks inside containers.
+* Add key_filename parameter to SSHChannel `issue#1639 <https://github.com/Parsl/parsl/issues/1639>`_
+* Repo now uses a Makefile to wrap several frequent operations such as:
+  * make test -> run the test-suite
+  * make install -> install Parsl
+* New `JobStatus` class for better monitoring of Jobs submitted to batch schedulers.
+* The `HighThroughputExecutor` no longer supports the `address` option and will raise an error when used.
+  The executor will by default use heuristics to detect and try all addresses. The addresses can be manually
```

Comment:
By default, the executor will use heuristics to detect and try all addresses. ...